package.json

```diff
@@ -52,10 +52,10 @@
     "clean": "rimraf ./peg_lib",
     "postclean": "mkdirp ./peg_lib",
     "prebuild": "npm-run-all clean",
-    "build:jsdoc": "frs-replace '^.*// [^/]*?no-jsdoc(?!-)[^/]*?$' '' -f gm -i './peg_src/jsdoctype.pegjs' --stdout | frs-replace '^.*// [^/]*?no-jsdoc-begin[\\s\\S]*?no-jsdoc-end.*?$' '' -f gm | pegjs --cache -o ./peg_lib/jsdoctype-jsdoc.js",
-    "build:typescript": "frs-replace '^.*// [^/]*?no-typescript(?!-)[^/]*?$' '' -f gm -i './peg_src/jsdoctype.pegjs' --stdout | frs-replace '^.*// [^/]*?no-typescript-begin[\\s\\S]*?no-typescript-end.*?$' '' -f gm | pegjs --cache -o ./peg_lib/jsdoctype-typescript.js",
-    "build:closure": "frs-replace '^.*// [^/]*?no-closure(?!-)[^/]*?$' '' -f gm -i './peg_src/jsdoctype.pegjs' --stdout | frs-replace '^.*// [^/]*?no-closure-begin[\\s\\S]*?no-closure-end.*?$' '' -f gm| pegjs --cache -o ./peg_lib/jsdoctype-closure.js",
-    "build:permissive": "pegjs --cache -o ./peg_lib/jsdoctype-permissive.js ./peg_src/jsdoctype.pegjs",
+    "build:jsdoc": "frs-replace '^.*// [^/]*?no-jsdoc(?!-)[^/]*?$' '' -f gm -i './peg_src/jsdoctype.pegjs' --stdout | frs-replace '^.*// [^/]*?no-jsdoc-begin[\\s\\S]*?no-jsdoc-end.*?$' '' -f gm | pegjs --allowed-start-rules TopTypeExpr,NamepathExpr,BroadNamepathExpr,ExternalNameExpr,ModuleNameExpr --cache -o ./peg_lib/jsdoctype-jsdoc.js",
+    "build:typescript": "frs-replace '^.*// [^/]*?no-typescript(?!-)[^/]*?$' '' -f gm -i './peg_src/jsdoctype.pegjs' --stdout | frs-replace '^.*// [^/]*?no-typescript-begin[\\s\\S]*?no-typescript-end.*?$' '' -f gm | pegjs --allowed-start-rules TopTypeExpr,NamepathExpr,BroadNamepathExpr --cache -o ./peg_lib/jsdoctype-typescript.js",
+    "build:closure": "frs-replace '^.*// [^/]*?no-closure(?!-)[^/]*?$' '' -f gm -i './peg_src/jsdoctype.pegjs' --stdout | frs-replace '^.*// [^/]*?no-closure-begin[\\s\\S]*?no-closure-end.*?$' '' -f gm| pegjs --allowed-start-rules TopTypeExpr,NamepathExpr,BroadNamepathExpr,ExternalNameExpr,ModuleNameExpr --cache -o ./peg_lib/jsdoctype-closure.js",
+    "build:permissive": "pegjs --allowed-start-rules TopTypeExpr,NamepathExpr,BroadNamepathExpr,ExternalNameExpr,ModuleNameExpr --cache -o ./peg_lib/jsdoctype-permissive.js ./peg_src/jsdoctype.pegjs",
```

Comment:
why does this allow 5 start rules when `ParseOptions` only allows 2?

Reply:
Good catch. Amended to update `ParserOptions` to allow all five as intended.